convert.py

```diff
@@ -27,5 +49,115 @@
 
         codelist_items.append(codelist_item)
 
+indent(template.getroot(), 0, 4)
 template.write('xml/FileFormat.xml', pretty_print=True)
 
+
+
+"""
+
+ISO Country Alpha 2
+
+"""
+
+XML_LANG = '{http://www.w3.org/XML/1998/namespace}lang'
+
+template = ET.parse('templates/Country.xml', ET.XMLParser(remove_blank_text=True))
+codelist_items = template.find('codelist-items')
+
+def add_code(code_text, country, withdrawn):
+    codelist_item = ET.Element('codelist-item')
+
+    if withdrawn:
+        codelist_item.attrib['withdrawn'] = withdrawn
+
+    code = ET.Element('code')
+    code.text = code_text
+    codelist_item.append(code)
+    
+    name = ET.Element('name')
+    codelist_item.append(name)
+    for short_name in country.findall('short-name'):
+        if XML_LANG in short_name.attrib:
+            narrative = ET.Element('narrative')
+            narrative.attrib[XML_LANG] = short_name.attrib[XML_LANG]
+            narrative.text = short_name.text
+            name.append(narrative)
+
+    description = ET.Element('description')
+    codelist_item.append(description)
+    for full_name in country.findall('full-name'):
+        if XML_LANG in full_name.attrib:
+            narrative = ET.Element('narrative')
+            narrative.attrib[XML_LANG] = full_name.attrib[XML_LANG]
+            narrative.text = full_name.text
+            description.append(narrative)
+
+    codelist_items.append(codelist_item)
+
+countries = ET.parse('source/iso_country_codes.xml')
```

Comment:
Looks as though the missing iso_country_codes.xml [comes from here](https://www.iso.org/obp/ui/#iso:pub:PUB500001:en)… Presumably this can’t be pulled in a completely automated way?